render/src/main/java/org/codice/imaging/nitf/render/imagerep/Rgb48ImageRepresentationHandler.java

```diff
@@ -20,10 +20,23 @@
 
 import javax.imageio.stream.ImageInputStream;
 
+import org.codice.imaging.nitf.core.image.PixelJustification;
+
 class Rgb48ImageRepresentationHandler extends AbstractRgbImageRepresentationHandler {
+    private final int bitsToDiscard;
+
     Rgb48ImageRepresentationHandler(final Map<Integer, Integer> bandMap,
-            final int actualBitsPerPixelPerBand) {
+            final int actualBitsPerPixelPerBand, final PixelJustification pixelJustification) {
         super(bandMap, actualBitsPerPixelPerBand);
+        if (actualBitsPerPixelPerBand - Byte.SIZE < 0) {
+            this.bitsToDiscard = 0;
```

Comment:
The codecov results show we are never hitting this line. (i.e. the test for `actualBitsPerPixelPerBand - Byte.SIZE < 0` is never true). Is that expected? I don't think it is valid in NITF, but if it was (say NBPP== 16, ABPP == 7) then perhaps we may need to shift depending on PixelJustification.